docs/development/docguide.rst

```diff
@@ -59,80 +56,60 @@ The details of the docstring format are described on a separate page:
 Sphinx Documentation Themes
 ===========================
 
-A custom Sphinx HTML theme is included in the `astropy-helpers`_ package.
-This allows the
-theme to be used by both Astropy and affiliated packages. This is done by
-setting the theme in the global Astropy sphinx configuration, which is imported
-in the sphinx configuration of both Astropy and affiliated packages.
-
-Using a different theme for ``astropy`` or affiliated packages
---------------------------------------------------------------
+An Astropy-branded Sphinx HTML theme is included in the astropy-sphinx-theme_
```

Comment:
Maybe just "An Astropy Project Sphinx HTML theme..."?  "Astropy-branded" sounds to me like we're trying to spread our brand around like to get ad revenue or something.

Reply:
Fixed